test/clipboard.vader

```diff
@@ -0,0 +1,60 @@
+# Test case file for yanking to system clipboard.
+# vim: set ft=vader
```

Comment:
Maybe also set the tabwidth and such using the mod line

Reply:
If you have Vader installed, I think it sets tab width and such automatically, through its ftdetect stage? I'll check that soon

Reply:
Possibly, though that would require contributors to have vader installed in the first place.
I'll add this as a dependency for development with a README commit, but I believe we should have the modline as failsafe anyway.

Reply:
Mm, good point. Especially since Vader complains if you don't get the formatting exactly right

Reply:
Done in e87f5e2